src/components/interaction-designer/Block.vue

```diff
@@ -365,6 +366,14 @@ export class Block extends mixins(Lang) {
     return this.isBlockEditorOpen && this.activeBlockId === this.block.uuid
   }
 
+  @flowNamespace.Action block_updateShouldShowBlockToolBar!: (
+    {blockId, value}: { blockId: string, value: boolean }
+  ) => void
+
+  get shouldShowBlockToolBar(): boolean {
+    return this.block?.vendor_metadata?.floip?.ui_metadata?.should_show_block_tool_bar ?? false
```

Comment:
I'm saving this `should_show_block_tool_bar` in block store, so it will be accessible by other component easily.